pom.xml

```diff
@@ -88,13 +88,13 @@
 
         <!--JAKARTA-->
         <jakarta.persistence.version>3.1.0</jakarta.persistence.version>
-        <jakarta.mail.version>2.1.0</jakarta.mail.version>
+        <jakarta.mail.version>2.1.1</jakarta.mail.version>
         <jakarta.servlet.version>6.0.0</jakarta.servlet.version>
         <jakarta.ws.rs.version>3.1.0</jakarta.ws.rs.version>
         <jakarta.xml.ws.version>4.0.0</jakarta.xml.ws.version>
         <jakarta.xml.bind.version>4.0.0</jakarta.xml.bind.version>
         <jakarta.transaction.version>2.0.1</jakarta.transaction.version>
-        <jakarta.activation.version>2.1.0</jakarta.activation.version>
+        <jakarta.activation.version>2.1.1</jakarta.activation.version>
```

Comment:
Is this expected? Is it caused by the migration script?

Reply:
@vmuzikar yes, it is.